### ISSUE TITLE
ARM64 release?

### ISSUE THREAD

**Issue by matthoosier** [audiamus/BookLibConnect#41]
Hi,

Thanks, this program is really useful. Would you consider making an ARM release of the installer?

**Comment by audiamus** [comment]
No, I don't have such plans at the moment. The big task at the moment is to integrate it into AAX Audio Converter.
